core-bundle/contao/themes/flexible/styles/main.css

```diff
@@ -55,6 +55,24 @@ body.popup {
 	background: var(--header-bg);
 }
 
+@keyframes slide-in-top {
+	0% {
+		transform: translateY(-40px);
+	}
+	100% {
+		transform: translateY(0);
+	}
+}
+
+.header--fixed #header {
+	position: fixed;
```

Comment:
+1 for using `position: sticky`, but I don‘t want to move the class back to the header, because depending on what people do with the controller, the class may need to be applied to multiple elements. It therefore makes sense to add it to the `<body>` element. Or even to the `<html>` element like modernizr.js does?

Reply:
Changed in fb6e6d9826e08d5f0b01958999b75986db6e403a.